script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -431,7 +439,7 @@ In that case, get up from the bench and come over here."
 [sha:c9ebcdde99e51d73e9420c81348f6c4af9c6169f]{
 -- Page 588, Offset 25796.
 -- 感情のない声で、アルクェイドはそう言った。
-Arcueid says that in a voice devoid of emotion.
+Arcueid's voice is cold.
```

Comment:
I prefer "devoid of emotion" over "cold" here for no particular reason

Reply:
I like how terse "cold" is. I wouldn't go so far as to say its a set phrase, but it is a motif. The terseness makes it all the more unfamiliar or surprising sounding

Reply:
np with that